Formula/certgraph.rb

```diff
@@ -0,0 +1,26 @@
+class Certgraph < Formula
+  desc "Intelligence tool to crawl the graph of certificate Alternate Names"
+  homepage "https://lanrat.github.io/certgraph"
+  url "https://github.com/lanrat/certgraph.git",
+    :tag => "20180217",
+    :revision => "41927387b1544a0d668c57b9883ed18381494ec4"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/lanrat/certgraph"
+    dir.install Dir["*"]
+
+    cd dir do
+      system "make", "dep"
+      system "make"
+      bin.install "certgraph"
```

Comment:
You'll likely need to add a `prefix.install_metafiles` line here.

This ensures things like `README`, `LICENSE` and `CHANGELOG` where available are installed into the prefix. Normally Homebrew handles this automatically with a little magic, but when you relocate the entire contents of the `buildpath` to another directory for the install process as is common with `go`-using formulae that logic falls on its face & it needs a little manual nudging. It's a known issue.